src/content/docs/accounts/accounts-billing/account-structure/multitenant-organizations.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Multi-tenant New Relic organizations
+tags:
+  - Accounts
+  - Account setup
+metaDescription: Options available for New Relic multi-tenant organizations, used by large enterprise customers and MSPs.  
+redirects:
+---
+
+Our multi-tenancy feature lets you have a New Relic organization that contains multiple sub-organizations within it. This is useful for large organizations that have departments they want to keep separate, and for managed service providers (MSPs) who need multiple tenant boundaries for their end users. 
+
+## Requirements 
+
+Requirements for our multi-tenancy feature: 
+
+* [Enterprise edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions)
+* Organizations must meet a few criteria to use this and must be approved: for more information, talk to your New Relic account representative
+
+## Details on organization structure [#org-structure]
+
+For an introduction to the basics of how New Relic organization and account structure works, see [Organizations](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure). 
+
+Here's a brief overview of the organization structure for our multi-tenant features:
+
+* The top-level New Relic organization is called the **managing organization**.
```

Comment:
Suggestion: All multi-tenant customers have a single managing organization and member organizations. 

Managing organizations have unique benefits compared to member organizations. These benefits include:

- Delegated Administration (insert doc) 
- Visibility into usage across organizations 
- Ability to administer member organizations 
... and more

For MSPs, the managing organization would be the MSP while each of the MSP's customers will be their own member organization.

Reply:
_Want to avoid using sub organization and top-level organization language for reasons mentioned above_